PokeAlarm/Events/MonEvent.py

```diff
@@ -83,6 +83,18 @@ def __init__(self, data):
         self.charge_duration = get_move_duration(self.charge_id)
         self.charge_energy = get_move_energy(self.charge_id)
 
+        # Catch Probs
+        self.catch_prob_1 = check_for_none(
+            float, data.get('catch_prob_1'), 'unkw')
+        self.catch_prob_2 = check_for_none(
+            float, data.get('catch_prob_2'), 'unkw')
+        self.catch_prob_3 = check_for_none(
+            float, data.get('catch_prob_3'), 'unkw')
+
+        # Attack Rating
+        self.rating_attack = data.get('rating_attack')
```

Comment:
`Unknown` and `check_for_none` here as well.

Reply:
done